flexibleengine/provider.go

```diff
@@ -328,12 +329,13 @@ func Provider() *schema.Provider {
 			"flexibleengine_rds_instances":       rds.DataSourceRdsInstances(),
 			"flexibleengine_rds_storage_types":   rds.DataSourceStoragetype(),
 
-			"flexibleengine_sfs_turbos":         sfs.DataSourceTurbos(),
-			"flexibleengine_smn_topics":         smn.DataSourceTopics(),
-			"flexibleengine_sms_source_servers": sms.DataSourceServers(),
-			"flexibleengine_vpc_route_table":    vpc.DataSourceVPCRouteTable(),
-			"flexibleengine_vpc_subnets":        vpc.DataSourceVpcSubnets(),
-			"flexibleengine_vpcs":               vpc.DataSourceVpcs(),
+			"flexibleengine_sfs_turbos":               sfs.DataSourceTurbos(),
+			"flexibleengine_smn_topics":               smn.DataSourceTopics(),
+			"flexibleengine_smn_message_templates_v2": smn.DataSourceSmnMessageTemplates(),
```

Comment:
it's better to define a new data source without version, just use `flexibleengine_smn_message_templates` will be fine.
The file name should also be updated.